Lib/test/test_posix.py

```diff
@@ -1522,6 +1522,7 @@ def test_pidfd_open(self):
         os.close(os.pidfd_open(os.getpid(), 0))
 
     @unittest.skipUnless(hasattr(os, "link"), "test needs os.link()")
+    @support.skip_android_selinux('hard links to symbolic links')
```

Comment:
For future reference, both of these lines could be replaced by `test.support.os_helper.skip_unless_hardlink`.

Reply:
Yes, I have found that all other tests for `os.link` are simply skipped if PermissionError was raised.